scripts/kafka.start.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+BASEDIR=$(dirname $0)
+source $BASEDIR/kafka.conf
+[[ -f $BASEDIR/kafka.local.conf ]] && source $BASEDIR/kafka.local.conf
+
```

Comment:
After checking the scripts: creating the Kafka directory is missing. 
other than that, it works :) 
![image](https://user-images.githubusercontent.com/57755873/200840964-770786b3-d141-49b5-ab6d-f096eb470622.png)

Reply:
I am not sure what exactly do you mean, but after you reboot all files in `/tmp` will be gone. Also systemd can clean up this directory, feel free to edit the config/kraft/server.properties file and use /var/lib/kafka or some other directory you want to keep.

You know what, let me move formatting into the start script, this way the directory will always be formatted.

Reply:
Not sure what are you commenting about. 
I tried to run the script but the Kafka directory was missing.
I created it manually, then ran the script and it worked.
just wanted to remind you to add the creation of the Kafka directory in the script :)